fv3core/tests/savepoint/translate/translate_grid.py

```diff
@@ -571,12 +571,14 @@ class TranslateUtilVectors(ParallelTranslateGrid):
     def __init__(self, grids, namelist, stencil_factory):
         super().__init__(grids, namelist, stencil_factory)
         self.max_error = 3e-12
-        self.near_zero = 3e-14
+        self.near_zero = 2e-6
```

Comment:
Is this applied to all variables in this savepoint test? or is it just the ones specified in the override file under ignore_near_zero_errors? There's also an option to specify each variable with its own near zero value, and I think it would be safer: https://github.com/ai2cm/pace/tree/main/fv3core/tests/savepoint/translate/overrides#variable-specific-near-zero-value

Reply:
I reduced the near-zero theshold, so this shouldn't be an issue now (I hope)